temporalcli/commands.workflow_reset.go

```diff
@@ -0,0 +1,205 @@
+package temporalcli
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	"go.temporal.io/api/common/v1"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+)
+
+var (
+	resetTypesMap = map[string]interface{}{
```

Comment:
Unused

Reply:
Looks like I managed to remove that as well, excellent. Removed in the next PR